src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -62,6 +64,24 @@ const {
   Deletion,
 } = ReactTypeOfSideEffect;
 
+function transferRef(fiber: Fiber, element: ReactElement<any>) {
+  if (typeof element.ref === 'string' && element._owner) {
```

Comment:
If this is a string and there is no owner, we should probably not set the fiber.ref to a string. Maybe move the second condition inside.